main.js

```diff
@@ -0,0 +1,13 @@
+let link = document.querySelector('li','ul')
+link.remove();
+
+let span = document.querySelector('span')
+span.remove();
+
+let posts = document.querySelector('.posts article')
+posts.remove();
+
+let header = document.querySelector('main .posts h3');
+header.remove();
+let index = document.querySelector('main .posts h3');
+index.remove();
```

Comment:
```suggestion
let allh3 = document.querySelectorAll('main .posts h3');
for (let h3 of allh3) {
h3.remove()
}
```
Instead of deleting the h3s one at a time, select all of them at once with `querySelectorAll`. `querySelectorAll` returns a **_node list_**. A node list is similar to an array, and .remove does NOT work on entire node lists. Instead, you have to iterate over the node list and delete each element one at a time.